src/tasks/installers/olm.ts

```diff
@@ -136,6 +136,17 @@ export class OLMTasks {
           task.title = `${task.title}...done.`
         }
       },
+      {
+        title: 'Change operator image',
+        enabled: () => !!flags['che-operator-image'],
+        task: async (_ctx: any, task: any) => {
+          const csvList = await kube.getClusterServiceVersions(flags.chenamespace)
+          const csv = csvList.items[0]
+          const jsonPatch = [{ op: 'replace', path: '/spec/install/spec/deployments/0/spec/template/spec/containers/0/image', value: flags['che-operator-image'] }]
+          await kube.patchClusterServiceVersion(csv.metadata.namespace!, csv.metadata.name!, jsonPatch)
+          task.title = `${task.title}... changed to ${flags['che-operator-image']}.`
```

Comment:
Maybe like usual `task.title = `${task.title}...done` I think ff we will get clear title above, than `done` will be good..

Reply:
With current code we'll see the image which will be used. If you think that such information is not needed - please let me know and I'll change it to just `done`.

Reply:
I don't think it is needed, since user use `--che-operator-flag` and provide the image explicitly